packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -126,6 +128,36 @@ function getWorkerToDurableObjectClassNamesMap(
 	return workerToDurableObjectClassNamesMap;
 }
 
+function getWorkerToWorkflowEntrypointClassNamesMap(
+	workers: Array<Pick<WorkerOptions, 'workflows'> & { name: string }>,
+) {
+	const workerToWorkflowEntrypointClassNamesMap = new Map(
+		workers.map((workerOptions) => [workerOptions.name, new Set<string>()]),
+	);
+
+	for (const worker of workers) {
+		for (const value of Object.values(worker.workflows ?? {})) {
+			if (value.scriptName) {
```

Comment:
nit: you're not testing this if branch, it would be great to do so, do you think it'd add too much complexity to the playground app?

Reply:
We can create a second playground like we do for Durable Objects to test cross-worker Workflows.

Reply:
b5ec843